### PR TITLE
making cahnges to resolve conflicts

### PR DESCRIPTION
**Pull request by sivagurubilli**
1) making changes across files to resolve errors
2) compleate zoom validation using zoom regex

### CODE REVIEW

File: src/Pages/AdminSidePages/AdminSignup/Captcha.tsx [hunk 1]
```diff
@@ -11,14 +11,14 @@ import {
...
-  username: string[];
+  username: string;
   captchaMatch: boolean;
 }
 
 //captcha function goes here
 const Captcha = ({ setCaptcha1 }: any) => {
   const [user, setUser] = useState<User>({
-    username: [],
+    username: "",
```
Comment (mangesh0712) [inline src/Pages/AdminSidePages/AdminSignup/Captcha.tsx:21]: rename the variable (relevant naming )

File: src/Pages/AdminSidePages/AdminSignup/index.tsx [hunk 2]
```diff
@@ -73,6 +74,8 @@ const initialCaptcha: Icaptchamatched = {
 //signup for admin component start here
 export default function AdminSignup() {
   const [signupState, setSignupState] = useState(initialValues);
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [body, setBody] = useState<string>("");
```
Comment (mangesh0712) [inline src/Pages/AdminSidePages/AdminSignup/index.tsx:78]: what is body ?? name the variable properly

File: src/Pages/AdminSidePages/AdminSignup/index.tsx [hunk 3]
```diff
@@ -237,6 +240,7 @@ const [isLoading,setLoading] = useState(false)
 
             <Box w="100%" mt="20px">
               <Captcha setCaptcha1={setCaptcha1} />
+              <CommonModalComponent isOpen={isOpen} setIsOpen={setIsOpen} body={body} />
```
Comment (mangesh0712) [inline src/Pages/AdminSidePages/AdminSignup/index.tsx:243]: body ??

File: src/Pages/StudentSidePages/StudentDashboard/index.tsx [hunk 4]
```diff
@@ -1,13 +1,21 @@
...
+import { GetDashboardLecturesService } from "../../../Services/LectureServices";
+import { ILectureResponse } from "@/Services/LectureInterface";
 
 // this component displays student side dashboard
 const Dashborad = () => {
-  const [dashboardLectures,setDashboardLectures] = useState()
+  const [dashboardLectures,setDashboardLectures] = useState<ILectureResponse[]>()
+
+  useEffect(()=>{
+    GetDashboardLecturesService().then((res)=>{
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentDashboard/index.tsx:15]: useEffect(()=>{
    GetDashboardLecturesService().then((res)=>{
      setDashboardLectures(res)
    })
  },[])

use below 


  useEffect(async()=>{
     const response = await  GetDashboardLecturesService()
     setDashboardLectures(response)
  },[])

File: src/Pages/AdminSidePages/AdminSignup/index.tsx (hunk 3 above)
Comment (sivagurubilli) [reply src/Pages/AdminSidePages/AdminSignup/index.tsx]: ok changed names

File: src/Pages/StudentSidePages/StudentDashboard/index.tsx (hunk 4 above)
Comment (sivagurubilli) [reply src/Pages/StudentSidePages/StudentDashboard/index.tsx]: use async await instead of .then

**Pull request merged**